locale/en/docs/es6.md

```diff
@@ -10,7 +10,7 @@ All ECMAScript 2015 (ES6) features are split into three groups for **shipping**,
 
 * All **shipping** features, which V8 considers stable, are turned **on by default on Node.js** and do **NOT** require any kind of runtime flag.
 * **Staged** features, which are almost-completed features that are not considered stable by the V8 team, require a runtime flag: `--es_staging` (or its synonym, `--harmony`).
```

Comment:
Well, `--es_staging` and `--harmony` are no synonyms. I think we should drop mentioning `--harmony` altgoldner, as is was the initial code name for ES6 and `--es_staging` already covers ES7 features as well.